mslib/msui/sideview.py

```diff
@@ -39,9 +39,11 @@
 from mslib.utils import thermolib
 from mslib.utils.config import config_loader
 from mslib.utils.units import units, convert_to
+from mslib.msui import autoplot_dockwidget as dock
 
```

Comment:
ToDo: similiar to the other names, not dock but may be apd, because  ad is occupied

Reply:
https://github.com/Open-MSS/MSS/pull/2548